nym-vpn-core/crates/nym-vpnd/src/main.rs

```diff
@@ -264,12 +268,35 @@ fn init_sentry() -> Option<ClientInitGuard> {
             traces_sample_rate: 1.0,
             enable_logs: true,
             shutdown_timeout: Duration::from_secs(2),
+            server_name: Some(Cow::Borrowed("nym")),
             ..Default::default()
         },
     ));
+    sentry::configure_scope(|scope| {
+        scope.set_tag("os_version", &os_info.os_version);
+        scope.set_tag("extra_metadata", os_info.extra.join(", "));
+        scope.set_user(Some(sentry::User {
+            id: Some(anonymize_identifier(&os_info)), // anonymized user identifier
+            ip_address: None,
+            ..Default::default()
+        }));
+    });
+
     Some(guard)
 }
 
+fn anonymize_identifier(os_info: &nym_vpn_lib::SysInfo) -> String {
+    let identifier = format!(
+        "{} {} {}",
+        os_info.os_version,
+        os_info.arch,
+        os_info.extra.join(" ")
```

Comment:
For now, I think it would be better to add the hostname as we do in tauri. Only relying on os version and info is kinda weak to identify a unique machine. Hostname will greatly improve that.

```
sysinfo::System::host_name().unwrap_or_else(|| "unknown".to_string()),
```